ekip/everykid/templates/base.html

```diff
@@ -51,8 +51,17 @@
             <a href="{% url 'about_ekip' %}">About Every Kid in a Park</a>
           </li>
           <li>
-            <a href="{% url 'privacy_policy' %}">Privacy policy</a>
+            <a href="https://www.doi.gov/privacy">Privacy policy</a>
```

Comment:
Should this go to our (simpler) privacy policy page? That has some information about kids and educators and then links out to the DOI policy? Does that make sense?